modules/admin-top-bar/assets/scss/_index.scss

```diff
@@ -0,0 +1,173 @@
+@import "../../../../assets/dev/scss/helpers/direction";
```

Comment:
Move to `modules/admin-top-bar/assets/scss/admin.scss`

Reply:
Done.